### PR TITLE
Ensure duplicated topic parameter does not break rendering (fix #769)

### LINKED ISSUE #769: TypeError: topic_search_for() got multiple values for keyword argument 'topic'
https://sentry.data.gouv.fr/share/issue/332e3935333534/

```
TypeError: topic_search_for() got multiple values for keyword argument 'topic'
(2 additional frame(s) were not displayed)
...
  File "udata/core/topic/views.py", line 67, in datasets
    **kwargs)

TypeError: topic_search_for() got multiple values for keyword argument 'topic'
```

### PR DESCRIPTION
**Pull request by noirbizarre**
See #769

### CODE REVIEW

File: udata/tests/frontend/test_topic_frontend.py [hunk 1]
```diff
@@ -56,6 +56,15 @@ def test_render_datasets_empty(self):
 
         self.assertEqual(len(self.get_context_variable('datasets')), 0)
 
+    def test_render_datasets_with_topic_param(self):
+        '''Should render a topic datasets page even with a topic parameter'''
+        self.init_search()
+        topic = TopicFactory(tags=['tag'])
+
+        url = url_for('topics.datasets', topic=topic) + '?topic=whatever'
```
Comment (davidbgk) [inline udata/tests/frontend/test_topic_frontend.py]: Why not `url_for('topics.datasets', topic=topic, qs={'topic': 'whatever'})`?

File: udata/tests/frontend/test_topic_frontend.py [hunk 2]
```diff
@@ -79,3 +88,12 @@ def test_render_reuses_empty(self):
         self.assert200(response)
 
         self.assertEqual(len(self.get_context_variable('reuses')), 0)
+
+    def test_render_reuses_with_topic_parameter(self):
+        '''Should render a topic reuses page even with a topic parameter'''
+        self.init_search()
+        topic = TopicFactory(tags=['tag'])
+
+        url = url_for('topics.reuses', topic=topic) + '?topic=whatever'
```
Comment (davidbgk) [inline udata/tests/frontend/test_topic_frontend.py]: Same here.

**Comment by noirbizarre** [comment]
Because I forgot ;)
Fixed

**Comment by davidbgk** [comment]
My main comment remains, I don't get why we have to that in the first place. Wrong links from our side? I can't find the referer.

**Comment by davidbgk** [comment]
Alright I get an example:

https://www.data.gouv.fr/fr/topics/international-et-europe/datasets?sort=-created&geozone=fr%2Fdistrict%2F043&license=fr-lo&topic=International+et+Europe&tag=equipement-sportif

I think we should not generate those links instead of fixing it later.

**Comment by noirbizarre** [comment]
Because we have a referral from Google with `topic=Something&format=something`.
(Referral is veisible from the full Sentry issue)

**Comment by noirbizarre** [comment]
If we produce these links I'l need more than an URL to fix this part too.
(but as some external links are coming with theses parameters, the late part is required too)

**Comment by noirbizarre** [comment]
Before I merge, have you seen somewhere in the where we provide these URLs ?

**Comment by davidbgk** [comment]
(Un)fortunately no, probably a remaining indexed link from an old bug :(

**Comment by noirbizarre** [comment]
OK, sad to have to handle this case :/

**Pull request merged**